lib/option.js

```diff
@@ -79,22 +80,35 @@ function defaultOptions(options, defaults) {
 }
 
 /**
- * Sanitize a URL, ensuring it has a scheme. If the URL begins with a slash or a period,
- * it will be resolved as a path against the current working directory. If the URL does
- * begin with a scheme, it will be prepended with "http://".
+ * Sanitize a URL, ensuring it has a scheme. If the URL begins with a slash or a period
+ * or it is a valid path relative to the current directory, it will be resolved as a
+ * path against the current working directory. If the URL does begin with a scheme, it
+ * will be prepended with "http://".
  * @private
  * @param {String} url - The URL to sanitize.
  * @returns {String} Returns the sanitized URL.
  */
 function sanitizeUrl(url) {
-	if (/^\//i.test(url)) {
-		return `file://${url}`;
+	let sanitizedUrl = url;
+	if (/^[./]/i.test(url) || fileExists(url)) {
+		sanitizedUrl = `file://${path.resolve(process.cwd(), url)}`;
+	} else if (!/^(https?|file):\/\//i.test(url)) {
+		sanitizedUrl = `http://${url}`;
 	}
-	if (/^\./i.test(url)) {
-		return `file://${path.resolve(process.cwd(), url)}`;
-	}
-	if (!/^(https?|file):\/\//i.test(url)) {
-		return `http://${url}`;
+	return sanitizedUrl;
+}
+
+/**
+ * Check whether a file exists on a given path without regard to permissions
+ * @private
+ * @param {String} filePath - the path to check
+ * @returns {Boolean} true if file exists, false if not
+ */
+function fileExists(filePath) {
+	try {
```

Comment:
You can use `fs.existsSync` and then drop the whole function

Reply:
I've attempted this, but tests fail.  I cannot explain why a call to any of the `fs` functions (existsSync or accessSync) produce errors when they are not captured within a try/catch.
If I call them directly, I get
```
TypeError: fs.existsSync is not a function
```
try rewriting the above function to call any function you desire within a try/catch and it will succeed. Call it directly, and it fails.  I have no explanation, but I'm sure someone here knows something about the *sync functions who could shed light on this.

Reply:
It needs to be stubbed out in the mocks first, I think.

https://github.com/pa11y/pa11y/blob/c102611acde655e79043995f5ed73345024340a5/test/unit/mock/fs.mock.js#L6